acestep/api/http/release_task_request_builder_test.py

```
@@ -134,6 +134,35 @@ def test_build_request_forwards_audio_code_string_and_cover_noise_strength(self)
         self.assertEqual("<|audio_code_7|>", request.audio_code_string)
         self.assertAlmostEqual(0.6, request.cover_noise_strength)
 
+    def test_build_request_forwards_session_retake_fields(self):
+        """Builder should include session-backed retake request fields."""
+        parser = _FakeParser(
+            {
+                "repaint_mode": "most natural",
+                "source_session_dir": "/tmp/source-session",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/source-session"

(S108)